SwordImportExportPlugin.inc.php

```diff
@@ -35,7 +35,8 @@ public function __construct($parentPlugin) {
 	public function register($category, $path, $mainContextId = null) {
 		$success = parent::register($category, $path, $mainContextId);
 		$this->addLocaleData();
-		HookRegistry::register('publishedarticledao::getAdditionalFieldNames', array($this, 'getAdditionalFieldNames'));
+		// see branch 3-1-2 if saving the deposit receipt is required
```

Comment:
Feel free to remove dead code, since it looks like we'll be releasing these changes for 3.2+ rather than 3.1.2.